src/decision/src/LidarDecision.cpp

```diff
@@ -31,6 +32,143 @@ void LidarDecision::scanCallBack(const sensor_msgs::LaserScan::ConstPtr& raw_sca
     // Deal with new messages here
 }
 
+
+
+
+
+double pi = 3.1415926;
```

Comment:
if you `include <cmath.h>;` in the header file, you should probably use `M_PI` instead, which should be more accurate. Also, for future notice, constants like this should be declared in the header file.